### PR TITLE
Add _33 Keyboard folder

### PR DESCRIPTION
**Pull request by tominabox1**
## Description

Adding new _33 keyboard designed by myself

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/_33/_33.c [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "_33.h"
```
Comment (fauxpark) [inline keyboards/_33/_33.c]:
```suggestion
#include "_33.h"
```

File: keyboards/_33/_33.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "quantum.h"
+#include "led.h"
```
Comment (fauxpark) [inline keyboards/_33/_33.h]:
```suggestion
```

File: keyboards/_33/_33.h [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+#define XXX KC_NO
+
+#define LAYOUT_33_BIG_SPACE( \
```
Comment (fauxpark) [inline keyboards/_33/_33.h]:
```suggestion
#define LAYOUT_33_big_space( \
```

File: keyboards/_33/_33.h [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+       K31,           K34,                K38       \
+) \
+{ \
+  { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09  }, \
+  { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19  }, \
+  { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29  }, \
+  { XXX, K31, XXX, XXX, K34, XXX, XXX, XXX, K38, XXX  }  \
+}
+
+#define LAYOUT_33_SPLIT_SPACE( \
```
Comment (fauxpark) [inline keyboards/_33/_33.h]:
```suggestion
#define LAYOUT_33_split_space( \
```

File: keyboards/_33/config.h [hunk 5]
```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#define CONFIG_H
+
```
Comment (fauxpark) [inline keyboards/_33/config.h]:
```suggestion
```

File: keyboards/_33/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (fauxpark) [inline keyboards/_33/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/_33/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+#include "_33.h"
```
Comment (fauxpark) [inline keyboards/_33/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/_33/rules.mk [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp 
+BOOTLOADER = amtel-dfu
```
Comment (fauxpark) [inline keyboards/_33/rules.mk]:
```suggestion
BOOTLOADER = atmel-dfu
```

**Comment by tominabox1** [comment]
I have made the requested changes and updated the branch.

File: keyboards/underscore33/config.h [hunk 9]
```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x6a50
```
Comment (drashna) [inline keyboards/underscore33/config.h]:
```suggestion
#define PRODUCT_ID 0x6A50
```

File: keyboards/underscore33/config.h [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+#define DEVICE_VER 0x0001
+#define MANUFACTURER tominabox1
+#define PRODUCT underscore33
+#define DESCRIPTION "Itty Bitty otho _30"
```
Comment (drashna) [inline keyboards/underscore33/config.h]: This isn't actually used

```suggestion
```

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**